bot/cogs/fun.py

```diff
@@ -87,6 +87,40 @@ async def eight_ball(self, ctx, *, question=None):
                 description="Usage: `.8ball will this command work?`",
             )
             await ctx.send(embed=embed)
+            
+    @command(
+        brief="Play a game of rock paper scissors",
+        description="Enter an option between rock, paper, scissor after .play",
+        aliases= ["play", "Play", "RPS"],
+    )
+    async def rps(ctx, *, response=None):
+        if isinstance(response, str):
+            response = response.lower()
+        options = ['rock', 'paper', 'scissors']
```

Comment:
it should be checked if `response in options`, this is where we should call `else:
            return await ctx.send("Please choose between rock, paper or scissors")` so that we do not waste any of the bot's time checking all the other if statements if we used an invalid input.